### DISCUSSION TITLE
doris用mysql驱动执行executeQuery(sql),返回的Field没有库表信息

### DISCUSSION THREAD

**Discussion by WeiPromise** [apache/doris#37447] [A - General / Q&A]
doris用mysql驱动执行executeQuery(sql),返回的Field没有库表信息，
<img width="548" alt="image" src="https://github.com/apache/doris/assets/36296620/e49e8ba5-f0c6-4d15-ad4d-aa93de642854">

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提个帖子
